src/tools/wait.ts

```diff
@@ -39,7 +39,9 @@ const wait = defineTool({
     const code: string[] = [];
 
     if (params.time) {
-      code.push(`await new Promise(f => setTimeout(f, ${params.time!} * 1000));`);
```

Comment:
all you needed to do was to replace code.push calls that were going nowhere with result.addCode.

also please add tests for generated code